i18n.php

```diff
@@ -652,15 +668,15 @@
   ),
   'ja' => 
   array (
-    'and' => 'かつ<|*',
+    'and' => 'かつ<|且つ<|*',
     'background' => '背景',
-    'but' => 'しかし<|ただし<|但し<|*',
+    'but' => 'しかし<|ただし<|但し<|然し<|*',
     'examples' => 'サンプル|例',
     'feature' => 'フィーチャ|機能',
     'given' => '前提<|*',
     'name' => 'Japanese',
     'native' => '日本語',
-    'rule' => 'Rule',
+    'rule' => 'ルール',
```

Comment:
Changes to the `rule` keyword will not actually affect our users at this point, since we don't support parsing `Rule` anyway.